collect_app/src/main/java/org/odk/collect/android/configure/SettingsImporter.kt

```diff
@@ -75,4 +84,20 @@ class SettingsImporter(
             }
         }
     }
+
+    private fun importProjectDetails(projectJson: JSONObject, projectId: String) {
+        val project = projectsRepository.get(projectId)!!
```

Comment:
Could we pass a `Project.Saved` into `fromJSON` to avoid the `!!` here? We should always have a project when doing an import.

Reply:
Yes we can do that. Done!